components/Pagination.vue

```diff
@@ -3,19 +3,20 @@
 		<ul class="pagination-list">
 			<li>
 				<nuxt-link v-if="currentPageNum > 1" :to="beforeLink" class="pagination-previous">
-					<Previous class="prev-next" />
+					<Previous class="previous" />
 				</nuxt-link>
 			</li>
 			<li v-if="currentPageNum > 2">
 				<nuxt-link :to="firstLink" class="pagination-link">
 					1
 				</nuxt-link>
 			</li>
-			<li v-if="before - 1 > 1">
-				<span class="pagination-ellipsis pagination-link">&hellip;</span>
+			<li v-if="before - 1 > 1" class="pagination-ellipsis">
+				<span>&hellip;</span>
 			</li>
-			<li v-if="currentPageNum > 1">
-				<nuxt-link :to="beforeLink" class="pagination-link">
+			<!-- when the current page is 2, this link points to page 1, which needs to show on the mobile style -->
+			<li v-if="currentPageNum > 1" v-bind:class="{'hide-on-mobile': hideProceedingPageButton}">
+				<nuxt-link :to="beforeLink" :aria-label="`Goto page ${before}`" class="pagination-link">
```

Comment:
As to [this comment](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/133#discussion_r403128921), `aria-label` for page numbers are not necessary. Please remove.